CITATION.cff

```diff
@@ -0,0 +1,77 @@
+cff-version: 1.2.0
+title: Kokkos
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
```

Comment:
https://kokkos.org/kokkos-core-wiki/citation.html says to cite the paper instead of the software artifact. Hence, I would rather do something similar to https://github.com/dealii/dealii/blob/master/CITATION.cff.